src/main/java/com/utilities/FileReader.java

```diff
@@ -0,0 +1,14 @@
+package com.utilities;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileReader {
+
+    public static byte[] readJsonFile(String path) throws IOException {
+
+        return Files.readAllBytes(Paths.get(path));
```

Comment:
Is it worth having this class for a single method which performs a single line of code? Being static, this will never be 'mock-able', either

Reply:
Deleted this class as this is only a one liner